src/ssl.c

```diff
@@ -43358,10 +43395,7 @@ void* wolfSSL_SESSION_get_ex_data(const WOLFSSL_SESSION* session, int idx)
 
 /* Note: This is a huge section of API's - through
  *       wolfSSL_X509_OBJECT_get0_X509_CRL */
-#if defined(OPENSSL_ALL) || (defined(OPENSSL_EXTRA) && \
-    (defined(HAVE_STUNNEL) || defined(WOLFSSL_NGINX) || \
-    defined(HAVE_LIGHTY) || defined(WOLFSSL_HAPROXY) || \
-    defined(WOLFSSL_OPENSSH) || defined(HAVE_SBLIM_SFCB)))
+#if defined(OPENSSL_EXTRA)
```

Comment:
I wouldn't do this, because it will increase the code size for people using `OPENSSL_EXTRA` without `OPENSSL_ALL` and without one of `HAVE_STUNNEL`, `WOLFSSL_NGINX`, etc.

Reply:
I understand but what is the logic of adding this section only when `OPENSSL_EXTRA` is used with one of the project defines (many of which define `OPENSSL_EXTRA` anyway). I need some things from this section for the `ca_names` handling. Alternatively, I can hide that stuff behind an `OPENSSL_ALL` guard.

Reply:
My guess at the logic would be that we don't want the code included for users just using `OPENSSL_EXTRA`, without any of  the items from the list of projects (OpenSSH, stunnel, etc.). The goal being reducing code size for that use case. Is it an important use case? I have no idea, so maybe the change is fine. @cconlon do you have any thoughts?

Reply:
I changed this back to how it was. I moved the `wolfSSL_sk_X509_NAME_*` functions outside of this boundary into an `OPENSSL_EXTRA` block. These are needed for the `client_ca` functions.

Reply:
Without these functions we are re-implementing a lot of the stack manipulation code.